unittests/test_pop_views_assembly/test_models.py

```diff
@@ -0,0 +1,110 @@
+
+# Copyright (c) 2017-2019 The University of Manchester
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+import spynnaker8 as sim
+from p8_integration_tests.base_test_case import BaseTestCase
+
+
+class TestPopulation(BaseTestCase):
```

Comment:
Inherit from unittest.TestCase
BaseTestCase is only needed for intergation tests that have to pickup cfg files, reset simulators ect

Reply:
oops BaseTestCase is correct here